tests/handlers/test_profile.py

```diff
@@ -344,6 +347,11 @@ def test_avatar_fetching_metadata_right_source(self) -> None:
                 user_id=UserID.from_string("@whatever:test"),
             )
         )
+        res = self.get_success(
+            self.handler.check_avatar_size_and_mime_type("mxc://test/local")
+        )
+        self.assertTrue(res)
+
         self.get_success(
```

Comment:
```suggestion
        # Now also check it for media on remote servers with explicit ports.
        #
        # Note that `check_avatar_size_and_mime_type` never actively pulls
        # remote media, but instead relies on it already having been cached.
        # So, for the test, it is adequate to cache the remote media, and then
        # check that it can be found. 
        self.get_success(
```